### ISSUE TITLE
Initial T0 unneeded and a pain for IDEX owners

### ISSUE THREAD

**Issue by tlhintoq** [Ultimaker/Cura#8379]
One of the great features of a IDEX printer such as the Tenlog TL-D3 (Hictop D3Hero) is that you can put it in Duplicate or Mirror modes and have the two extruders act like their own printers; make two of a thing at the same time.

Cura however has the need to put a T0 command at the start of the exported gCode.  Why?  T0 is the default anyway and all commands are going their if you don't specify an extruder.

However by adding it, it screws up the behavior of dual head machines.  If I put my machine in 'Duplicate' mode then run the gCode as saved by Cura, it only heats the first extruder.  If I put it in "Extruder 2" mode this T0 flips it right back to Extruder 1.

Basically every file I export out of Cura needs to then be opened in Notepad++ and this redundant initializer deleted.
Its not difficult.  But its needless extra work for the user.  If you could simply not go out of your way to add something that's not needed to begin with, that would be awesome!

![2020-09-13_15-52-36](https://user-images.githubusercontent.com/11351506/93027388-71414500-f5da-11ea-8095-3a43d8de8e0f.jpg)

**Comment by Ghostkeeper** [comment]
> Cura however has the need to put a T0 command at the start of the exported gCode. Why? T0 is the default anyway and all commands are going their if you don't specify an extruder.

Many printers remember their state after a print. If the last print ended up on T1, it will stay on T1 and you'd be using the wrong extruder for your print.

The `T0` is switching to the `initial_extruder_nr`, so if your print starts with T1 it would switch to T1.

As a workaround you can add a search-and-replace post-processing script that replaces `;Generated with Cura_SteamEngine 4.7.1\nT0` by just `;Generated with Cura_SteamEngine 4.7.1`. That should prevent the need for manually editing every g-code file.

I'll close this as a duplicate of https://github.com/Ultimaker/Cura/issues/7395.
